ando/checker.py

```diff
@@ -32,7 +32,7 @@ def main():
             directory = args.path
         except IndexError:
             directory = '.'
-        error = is_AnDO_verbose(directory)
+        error = is_AnDO_verbose(directory,True)
```

Comment:
Please add a space between the two arguments of your function: (directory, True)

Reply:
Fixed

Reply:
Fixed